skrub/_drop_column_if_null.py

```diff
@@ -25,8 +40,16 @@ def fit_transform(self, column, y=None):
         """
         del y
 
-        self.drop_ = sbd.is_all_null(column)
-
+        if self.threshold == 1.0:
```

Comment:
this doesn't match the docstring or the code below which drop the column if the proportion is strictly greater than the threshold. of course, the proportion cannot be strictly greater than 1.0 ...
we could say that we drop if it is greater or equal than the threshold but then we would have no way of expressing "drop if it contains any nulls" because threshold=0.0 would drop all columns

not sure what we should do about this, but in any case the docstring and tests should be very clear on what happens in the cases where the 2 values are equal

Reply:
I modified the code to try and address both this and the case when no nulls are found

Reply:
Could we make an exception for 0 and drop when the number of nulls is > 0? Not 100% consistent but this is the high level behaviour I would expect as a user

Reply:
That's how I coded it in the latest commit